repos/system_upgrade/el7toel8/actors/commonleappdracutmodules/files/dracut/85sys-upgrade-redhat/do-upgrade.sh

```diff
@@ -78,13 +81,27 @@ save_journal() {
     # Q: would it be possible that journal will not be flushed completely yet?
     echo "writing logs to disk and rebooting"
 
-    local logfile="$NEWROOT/var/log/leapp/leapp-upgrade.log"
+    local logfile="/sysroot/tmp-leapp-upgrade.log"
+
+    # Create logfile if it doesn't exist
+    [ -n $logfile ] && > $logfile
+
+    # If file exists save the journal
+    if [ -e $logfile ]; then
+        # Add a separator
+        echo "### LEAPP reboot ###" > $logfile
```

Comment:
```suggestion
        echo "### LEAPP reboot ###" >> $logfile
```
Not sure if there is any chance for anything to be there yet...

Reply:
not in this one, and I want it to be empty.